src/System.CommandLine/ArgumentArity.cs

```diff
@@ -47,7 +47,7 @@ public ArgumentArity(int minimumNumberOfValues, int maximumNumberOfValues)
 
             if (tokenCount < minimumNumberOfValues)
             {
-                if (symbolResult.UseDefaultValueFor(argument))
+                if (symbolResult!.UseDefaultValueFor(argument))
```

Comment:
Didn’t follow. If symbolResult is null, then argumentResult is null, then tokenCount is zero. Wouldn’t we get here with null symbolResult?